src/configs/commands.js

```diff
@@ -0,0 +1,10 @@
+const { ServerTime } = require("../useCases");
```

Comment:
an configuration should not call a use case, only interfaces should

![image](https://user-images.githubusercontent.com/20214425/220774841-7809c927-2691-4d29-b8d9-6851cc82c8b5.png)